src/browser/modules/Frame/styled.tsx

```diff
@@ -105,9 +106,9 @@ export const StyledFrameContents = styled.div<FullscreenProps>`
   min-height: ${dim.frameBodyHeight / 2}px;
   max-height: ${props =>
     props.fullscreen
-      ? '100vh'
+      ? 'auto'
       : dim.frameBodyHeight - dim.frameStatusbarHeight * 2 + 'px'};
-  ${props => (props.fullscreen ? 'height: 100vh' : null)};
+  ${props => (props.fullscreen ? 'height: calc(100vh - 40px)' : null)};
```

Comment:
Would it make sense to make the `40` value here a const, especially if it's also used to set the height of the title bar?

Reply:
Yes, even better would be to refactor the fullscreen stuff to not need these calc workarounds. After a quick demo /w design/pm we decided to skip the fullscreen until we can do so properly, so I've reverted the commit that adds these fullscreen changes.